src/versions.ts

```diff
@@ -324,6 +325,9 @@ export async function prepareVersionDirectory(
     path.join(installDir, 'package.json'),
     JSON.stringify(packageJson, null, 2)
   );
+  if (npmrc) {
+    await fsExtra.copySync(path.resolve(npmrc), path.join(installDir, '.npmrc'), {});
```

Comment:
Can use the async version of copy here (currently it's awaiting the return of the sync version). Also might as well use the `node:fs/promises` version now? We can probably drop fs-extra sometime.

Reply:
I updated to `fsExtra.copy()`. It feels like moving a bunch of stuff over to `node:fs/promises` would be best done in a separate PR, but let me know if that's acceptable to get this PR move through or not.

Reply:
Yeah SGTM